### ISSUE TITLE
[DO] Validate BQ projects limits 

### ISSUE THREAD

**Issue by oleurud** [CartoDB/cartoframes#1504]
We need to validate:
- the limit is working
- we showing the proper message in CF

**Comment by Jesus89** [comment]
Is this ticket still valid with the new Enrichment API?

**Comment by oleurud** [comment]
Yes. @gonzaloriestra and I validated the first one (the limit is working by project) some time ago, but probably the error from Google will be lost in the API (although it will be in the logs)

**Comment by Jesus89** [comment]
https://app.clubhouse.io/cartoteam/story/93714/validate-bq-project-limits-check
